roofit/roofitcore/inc/RooAbsGenContext.h

```diff
@@ -82,7 +81,7 @@ class RooAbsGenContext : public TNamed, public RooPrintable {
   RooArgSet _protoVars;         ///< Prototype observables
   Int_t _nextProtoIndex;        ///< Next prototype event to load according to LUT
   RooAbsPdf::ExtendMode _extendMode ;  ///< Extended mode capabilities of p.d.f.
-  Int_t* _protoOrder ;          ///< LUT with traversal order of prototype data
```

Comment:
It looks like it is not transient. An I/O rule I think is needed to read old files

Reply:
This class has class version `0`, so we don't need to consider IO I think, right?